packages/tabs/src/TabTitle.tsx

```diff
@@ -143,6 +140,23 @@ const listTitle = css`
     outline: none;
   }
 
+  // We create a pseudo element that's the width of the bolded text
+  // This way there's no layout shift on hover when the text is bolded.
+  &:before {
+    content: attr(data-text);
+    content: attr(data-text) / '';
+    height: 0;
+    font-weight: 700;
+    visibility: hidden;
+    overflow: hidden;
+    user-select: none;
+    pointer-events: none;
+
+    @media speech {
```

Comment:
I actually wasn't sure – is the content CSS property read by screen readers? If so, good to know!

Reply:
It will be read out by screen readers yeah. Just noticed though that `@media screen` is [deprecated in MQ4](https://drafts.csswg.org/mediaqueries/#media-types). I'll update this appropriately

Reply:
Looking further `visibility: hidden` will do the same thing that the `@media speech` rule is trying to do

Reply:
Ahh perfect! Forgot about that.